src/main/java/Duke.java

```diff
@@ -1,10 +1,137 @@
+    public static void addTaskToList(Task[] listInput) {
+        //if list empty, inform user and await next command
+        if (taskCount == 0) {
+            System.out.println(underscoredLine + System.lineSeparator() + "\tThe list is empty." 
+            + System.lineSeparator() + underscoredLine);
+            return;
+        }
+        //if list non-empty, print out all existing tasks
+        System.out.println(underscoredLine + System.lineSeparator());
+        for (int i = 0; i < taskCount; i++) {
+            System.out.println("\t" + Integer.toString(i + 1) + "." + listInput[i].toString());
+        }
+        System.out.println(underscoredLine);
+        return;
+    }
+
+    public static void updateTaskDone(String taskNumberInput, Task[] listInput){
+        int queryNumber = Integer.parseInt(taskNumberInput);
+        boolean isOutOfRange = queryNumber < 1 || queryNumber > taskCount;
+        
+        //handle case where user inputs non-existing task number to mark as done
+        if (isOutOfRange){
+            System.out.println(underscoredLine + System.lineSeparator() + "\tInvalid task number."
+            + System.lineSeparator() + underscoredLine);
+            return;
+        }
+        //handle case where user tries to mark as done an already completed task
+        boolean isTaskAlreadyDone = listInput[queryNumber-1].getIsDone();
+        if (isTaskAlreadyDone){
+            System.out.println(underscoredLine + System.lineSeparator()
+            + "\tThis task has already been marked completed." + System.lineSeparator() + underscoredLine);
+            return;
+        }
+        listInput[queryNumber-1].markAsDone();
+        System.out.println(underscoredLine + System.lineSeparator() + "\tGreat job! I've marked this task as done:\n"
+        + "\t" + Integer.toString(queryNumber) + ".[" + listInput[queryNumber-1].getStatusIcon() + "] "
+        + listInput[queryNumber-1].getDescription() + System.lineSeparator() + underscoredLine);
+    }
+
+    private static void insertNewTask(Task[] taskList, String userInput, String[] tokenizedInput) {
+        Task newTask;
+        switch (tokenizedInput[0]) {
+        case ("todo"):
+            newTask = new Todo(splitUserInput(userInput)[0], splitUserInput(userInput)[1]);
+            break;
+        case ("deadline"):
+            newTask = new Deadline(splitUserInput(userInput)[0], splitUserInput(userInput)[1]);
+            break;
+        case ("event"):
+            newTask = new Event(splitUserInput(userInput)[0], splitUserInput(userInput)[1]);
+            break;
+        default:
+            newTask = new Task(userInput);
+            break;
+        }
+
+        taskList[taskCount] = newTask;
+        taskCount++;
+        System.out.println(underscoredLine + System.lineSeparator() + "\tGot it. I've added this task: \n\t"
+        + newTask.toString() + System.lineSeparator() + "\tNow you have " + taskCount + " tasks in the list.\n"
+        + underscoredLine);
```

Comment:
Consider shortening the line, into String s = ... Then do a few s += ... to split up the different component so that it is more readable and easier to understand. Finally then print the String s. Also, for wrapped text, consider indenting 8 spaces more than the parent line